guides/common/modules/proc_assigning-ansible-roles-to-an-existing-host.adoc

```diff
@@ -0,0 +1,22 @@
+[id="adding-ansible-roles-to-an-existing-host_{context}"]
+= Assigning Ansible Roles to an Existing Host
```

Comment:
It is not stated what does assigning mean. I think that a sentence saying that before running a role on a host you need to assign this role to the host would make it.

Reply:
Again this is not a change that I made and exists currently in the docs and out of scope for this PR! Next PR!